publish-config.yml

```diff
@@ -0,0 +1,30 @@
+project:
+    Name: gh-svn-pypi-publisher
+    Description: Example project for publishing to PyPI
+
+publishers:
+    name: providers
+    url: https://gh-svn-pypi-publisher/
+    path: "airflow/providers/"
+    version_pattern: '^(.*?)-(rc\d+-)?\d+'
+    extensions:
+      - .tar.gz
+      - .tar.gz.asc
+      - .tar.gz.sha512
+      - -py3-none-any.whl
```

Comment:
We Likely need to make it a bit more flexible and allow regexp here. While  Airlow only publishes one wheel, many of other projects that have some binary components will have multiple wheel files for many architectures/ ABIs - https://peps.python.org/pep-0491/#file-name-convention

Reply:
This is not hardcoded one, any file extension can be provided from the config file. Sure, agree can update with regex support. :)